src/applications/static-pages/static-pages-entry.js

```diff
@@ -124,11 +107,13 @@ if (burialPages.has(location.pathname)) {
   });
 }
 
-if (disabilityPages.has(location.pathname) && brandConsolidation.isEnabled()) {
+if (disabilityPages.has(location.pathname)) {
   createDisabilityIncreaseApplicationStatus(store);
 }
 
-if (location.pathname === '/disability-benefits/increase-claims-testing/') {
+if (
+  location.pathname === '/disability-benefits/apply/form-526-disability-claim/'
+) {
   create526EmailForm(store);
 }
 
```

Comment:
On this page, I tore down the feature flag, and removed any Vets.gov page paths. 

CC: @U-DON